_posts/2022-03-24-opensearch-java-runtime.markdown

```diff
@@ -0,0 +1,78 @@
+
+### No-JDK Distributions
+
+Releasing a no-JDK distribution of OpenSearch is tracked in [opensearch-build#99](https://github.com/opensearch-project/opensearch-build/issues/99). The current work-around is to download the existing distribution, remove the JDK and set `JAVA_HOME` or `OPENSEARCH_JAVA_HOME`. This is not ideal, so please add your +1 to the issue if you need this feature.
+
+### Platform Specifics
+
+#### FreeBSD
+
+FreeBSD packages are available for OpenSearch from [textproc/opensearch](https://www.freshports.org/textproc/opensearch/). These packages do not bundle a version of Java, and depend on one of the Java versions installed on FreeBSD.
+
+Users building their own packages can customize the version of Java the package will depend on by setting the java version in the [`DEFAULT_VERSIONS` environment variable](https://wiki.freebsd.org/Ports/DEFAULT_VERSIONS) when building, e.g. `DEFAULT_VERSIONS=java=15`. See [opensearch-build#101](https://github.com/opensearch-project/opensearch-build/issues/101) and [Releasing for FreeBSD](https://github.com/opensearch-project/opensearch-build#releasing-for-freebsd) for more information.
+
+#### Arch Linux
+
+The [community-contributed Arch Linux distribution of OpenSearch](https://wiki.archlinux.org/title/OpenSearch) has [been using](https://github.com/archlinux/svntogit-community/blob/packages/opensearch/trunk/PKGBUILD#L34) Java 11 for both versions 1.2.3 and 1.2.4.
+
+#### Windows and MacOS
+
+At this moment there's no official Windows or MacOs distribution of OpenSearch. However, this project does support building and assembling OpenSearch for Windows and MacOS, with some caveats. The version of JDK used and the configuration options are the same as on Linux. See [opensearch-build#33](https://github.com/opensearch-project/opensearch-build/issues/33), [#37](https://github.com/opensearch-project/opensearch-build/issues/37) and [#38](https://github.com/opensearch-project/opensearch-build/issues/38) for details and how you can contribute.
+
+### Documentation
+
+The [compatibility documentation](https://opensearch.org/docs/latest/opensearch/install/compatibility/) has been updated to reflect the above changes in [documentation-website#459](https://github.com/opensearch-project/documentation-website/pull/459).
```

Comment:
The blog post just kind of... ends. 

You need a call to action - what should the reader do next?